kratos/utilities/variable_utils.cpp

```diff
@@ -501,7 +501,7 @@ void VariableUtils::UpdateInitialToCurrentConfiguration(const ModelPart::NodesCo
     #pragma omp parallel for
     for (int i=0; i<num_nodes; i++) {
         const auto it_node  = nodes_begin + i;
-        noalias(it_node->GetInitialPosition()) = it_node->Coordinates();
+        noalias(it_node->GetInitialPosition().Coordinates()) = it_node->Coordinates();
```

Comment:
Regarding the compile error: the line above (486) compiles for me without the `.Coordinates()` on the RHS, but if I remove the `.Coordinates()` here i get:
```
/home/jcotela/Kratos/kratos/utilities/variable_utils.cpp:504:46:   required from here
/home/jcotela/Kratos/kratos/includes/amatrix_interface.h:484:73: error: cannot bind non-const lvalue reference of type ‘Kratos::Point&’ to an rvalue of type ‘Kratos::Point’
 template <typename T> T& noalias(T& TheMatrix){return TheMatrix.noalias();}
```

Reply:
The fix in this PR works, and removing the noalias works as well. It is not a problem of the construction, but rather of trying to do `noalias(point) = array_1d`.

Reply:
Thanks @jcotela!

What I don't understand how this works with ublas.... Maybe I need to define a reference expression....

Reply:
One thing that compiled for me (but I didn't test it, so it may have unintended consequences) is to explicitly cast within the `noalias` definition:
```C++
template <typename T> T& noalias(T& TheMatrix){return (T&)TheMatrix.noalias();}
```
(The only change with respect to master is the `(T&)` after the return).